### PR TITLE
Config

### PR DESCRIPTION
**Pull request by bmccallister**
Verified working now with other apps.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,9 +11,10 @@ let DOC = require('dynamodb-doc');
  */
 export default class {
   constructor(config) {
-    AWS.config.update(config);
+    // AWS.config.update(config);
```
Comment (stephen-ta) [inline src/index.js]: Don't really need this comment in here ;)

**Comment by bmccallister** [comment]
Fixed

**Comment by Fluidbyte** [comment]
:accept:

**Comment by stephen-ta** [comment]
:accept:

**Pull request merged**